docs/GettingStartedDocs/Windows_vscode.md

```diff
@@ -0,0 +1,117 @@
+# Building And Debugging Using Visual Studio Code
+
+This document provides a brief overview of how to build and debug Open Enclave applications using VS Code.
+
+## Install VS Code
+
+Latest version of Visual Studio Code can be installed from [https://code.visualstudio.com/](https://code.visualstudio.com/)
+
+
+## Install VS Code Extensions
+
+Install the following VS Code extensions. Click on an image to navigate to the Visual Studio Code Marketplace page for the extension.
```

Comment:
There's actually two separate VS Code Extensions right now.   It seems odd to mention one without mentioning the other.   Eventually they should be combined into the same one.   Not sure what to do about this right now as far as getting started docs, since there's two separate getting started docs right now.  Suggestions?

Reply:
> There's actually two separate VS Code Extensions right now. It seems odd to mention one without mentioning the other. Eventually they should be combined into the same one. 

@HernanGatta @radhikaj When will the OPTEE documentation be merged into master?
If there is documentation on OPTEE's vs code workflow, I can add a line at the start of this file "For developing Open Enclave TrustZone application using Visual Studio Code, see <link-to-optee-vs-code-doc>"

Reply:
@HernanGatta is preparing a PR for this.

Reply:
@dthaler Currently all the docs we have are SGX specific. When Getting Started docs/building instructions etc for Trustzone areintegrated with master, debugging instructions for Trustzone can also be included.